source/deprecations/v2.x.html.md

```diff
@@ -753,3 +753,92 @@ component.appendTo('.my-component-wrapper');
 ```
 
 Note that both APIs are private, so no public API is been deprecated here.
+
+
+#### `{{render` helper
+
+##### until: 3.0.0
+##### id: ember-template-compiler.deprecate-render
+
+Using the `{{render` helper is deprecated in favor of using components.
+Please refactor to a component and invoke thusly:
+
+For example, if you had:
+
+```hbs
+{{render 'my-sidebar'}}
+```
+
+```hbs
+{{! app/templates/my-sidebar.hbs }}
+<p>template stuff here</p>
+```
+
+```js
+// app/controllers/my-sidebar.js
+export default Ember.Controller.extend({
+});
+```
+
+Would be refactored to:
+
+```hbs
+{{my-sidebar}}
+```
+
+```hbs
+{{! app/templates/components/my-sidebar.hbs }}
+<p>template stuff here</p>
+```
+
+```js
+// app/components/my-sidebar.js
+export default Ember.Component.extend({
+});
+```
+
+#### Rendering into a {{render}} helper that resolves to an {{outlet}}.
+
+##### until: 3.0.0
+##### id: ember-routing.top-level-render-helper
+
+For example, if you had:
```

Comment:
It would be nice to explain what the functionality is before giving an example. I expect most developers are not familiar with this functionality, and if you don't explain what it is you'll give them palpitations trying to guess if their codebase needs to change :-)